CHANGELOG

```diff
@@ -1,3 +1,30 @@
+Release 97 Knossos (2017-11-16)
+-------------------------------
+Scala Common Enrich: add function to camelCase all JSON fields in Adaptor (#3538)
+EmrEtlRunner: add ability to skip RDB Loader consistency check (#3529)
+EmrEtlRunner: bump to 0.30.0 (#3526)
+EmrEtlRunner: uncompress gzipped raw files when copying to HDFS (#3525)
+Scala Common Enrich: remove toSecond parameter in Adapter (#3534)
+Spark Enrich: Fix failing tests due to dependency on new SCE (#3535)
+Spark Enrich: bump version to 1.11.0 (#3533)
+Spark Enrich: fix tests that fail with error when running on a platform that doesn't have native-lzo (#3508)
+Spark Enrich: fix tests that fail when running on an alternative iglu service (#3503)
+Common: rename AWS deployment credentials in .travis.yml (close #3115)
+Spark Enrich: improve error message in test to show index line (#3494)
+Scala Common Enrich: Default port to 443 if scheme is https (#3483)
+Scala Common Enrich: tolerate content type for GET requests sent to Clojure Collector (#2743)
+Scala Common Enrich: add adapter to pre-process Unbounce webhooks (#2615)
+Spark Enrich: add test for Unbounce Adapter (#2745)
+Spark Enrich: add test for StatusGator Adapter (#2722)
+Scala Common Enrich: add adapter to pre-process Statusgator webhooks (#2169)
+Spark Enrich: add test for Mailgun Adapter (#2763)
+Scala Common Enrich: add adapter to pre-process Mailgun webhooks (#2734)
+Spark Enrich: add test for Olark Adapter (#2792)
+Scala Common Enrich: add Adapter to pre-process Olark events (#1014)
+Scala Common Enrich: make enrichments.ExtractEventTypeSpec timezone-safe (#3481)
+Scala Common Enrich: bump user-agent-utils to 1.20 (#2930)
```

Comment:
Could you order the lines according to the project similar to what's done below? Thanks.